Tasks/DownloadSecureFileV1/task.json

```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 177,
+        "Minor": 178,
```

Comment:
Nit: Let's go with version 1.177.1 since we're going to hotfix this. We should also cherry pick this commit to `releases/m177` once its merged

Reply:
Yep, makes sense, thanks! Fixed it